apps/condo/domains/common/components/antd/Select.tsx

```diff
@@ -15,26 +12,20 @@ export interface CustomSelectProps<T> extends SelectProps<T> {
 }
 
 const Select = <T extends string | number | { value: any, label: any }> (props: CustomSelectProps<T>) => {
-    const { eventName: propEventName, eventProperties = {}, onChange, ...restProps } = props
+    const { eventName: propEventName, eventProperties = {}, onSelect, ...restProps } = props
     const { logEvent, getEventName } = useTracking()
 
     const eventName = propEventName ? propEventName : getEventName(TrackingEventType.Select)
     const componentProperties = { ...eventProperties }
 
-    const onChangeCallback: CustomSelectProps<T>['onChange'] = (value, option) => {
+    const onSelectCallback = useCallback((value, option) => {
         if (eventName) {
-            let selectedValue = null
-
-            if (isArray(option)) {
-                selectedValue = compact(option.map(opt => get(opt, 'title', false) || get(opt, 'label')))
-            } else {
-                selectedValue = get(option, 'title')
-            }
-
-            if (!isEmpty(selectedValue)) {
-                componentProperties['component'] = { value: selectedValue }
+            const componentValue = get(option, 'title')
 
+            if (componentValue) {
                 const componentId = get(restProps, 'id')
+                componentProperties['component'] = { value: componentValue }
```

Comment:
This will not work correctly with multiple select. For example, at the ticket filters

Reply:
Why? If we leave it as it was, then we will have incorrect statistics on clicks on items

**Example**:

1 click: [`option_1`]
2 click: [`option_1`, `option_2`]
3 click: [`option_1`, `option_2`, `option_3`]

If we make a selection only for `option_1`, then we will have 3 actions for this item. But in fact there was only one click on `option_1`. And the stats are wrong.

Reply:
What useful information can we extract from these arrays?

Reply:
> What useful information can we extract from these arrays?

Useful information here is what actually user see at the component, i.e. whatever is selected, not the instant event.
 
We do not pass an input event to the text field about each character. For this, input delay is used. 
Here we want to see the set of values that was in the value of this element at the time the event was sent.

Reply:
Why do we need to know intermediate values?

Reply:
This way we don't know which item was selected.